rero/rero-ils-ui
# package: upgrade to NodeJS 12

Co-Authored-by: Olivier DOSSMANN <git@dossmann.net>

## Why are you opening this PR?

To update Node version from 10 to 12.

## How to test?

If you **already have** rero-ils-ui directory:

```bash
cd rero-ils-ui
node --version # and keep the version, for example 10.16.3
nvm install lts/erbium --reinstall-packages-from=10.16.3 # replace here the version 10.16.3 by yours
nvm uninstall 10.16.3
npm i
npm ci
ng --version # to check all is right for you
```

You should obtain something like:

```bash

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 8.3.28
Node: 12.18.2
OS: linux x64
Angular: 8.2.14
... animations, common, compiler, compiler-cli, core, elements
... forms, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.25
@angular-devkit/build-angular     0.803.25
@angular-devkit/build-optimizer   0.803.25
@angular-devkit/build-webpack     0.803.25
@angular-devkit/core              8.3.25
@angular-devkit/schematics        8.3.28
@angular/cdk                      8.2.3
@angular/cli                      8.3.28
@angular/flex-layout              8.0.0-beta.27
@ngtools/webpack                  8.3.25
@schematics/angular               8.3.28
@schematics/update                0.803.28
rxjs                              6.4.0
typescript                        3.4.5
webpack                           4.39.2
```

**If it's your first rero-ils-ui installation**: follow [the quickstart guide to install rero-ils-ui](https://github.com/blankoworld/rero-ils-ui/tree/doo-upgrade-packages#quick-start)

## Code review check list
